### ISSUE TITLE
About the calculation of PSNR

### ISSUE THREAD

**Issue by dmhdmhdmh** [jaewon-lee-b/lte#16]
Hello, I'm sorry to bother you again. I am confused about the difference in psnr calculation between benchmark and div2k. Why do I multiply diff by graycoeffs when calculating psnr on benchmark datasets?

def calc_psnr(sr, hr, dataset=None, scale=1, rgb_range=1):
    diff = (sr - hr) / rgb_range
    if dataset is not None:
        if dataset == 'benchmark':
            shave = scale
            if diff.size(1) > 1:
                gray_coeffs = [65.738, 129.057, 25.064]
                convert = diff.new_tensor(gray_coeffs).view(1, 3, 1, 1) / 256
                diff = diff.mul(convert).sum(dim=1)
        elif dataset == 'div2k':
            shave = scale + 6
        else:
            raise NotImplementedError
        valid = diff[..., shave:-shave, shave:-shave]
    else:
        valid = diff
    mse = valid.pow(2).mean()
    return -10 * torch.log10(mse)

**Comment by jaewon-lee-b** [comment]
Hello @dmhdmhdmh
For deep learning-based super-resolution performance evaluation (including EDSR, RDN, SwinIR, and LIIF), we compute PSNR as follows after the EDSR paper.

1. DIV2K => RGB space PSNR
2. Benchmark dataset (Set5/14, B100, Urban100, Manga109) => Y space PSNR

**Comment by jaewon-lee-b** [comment]
Y space PSNR is useful for evaluating edge information...
